ui-common/src/components/EntryDetailed/EntrySections/EntrySections.tsx

```diff
@@ -117,6 +117,37 @@ interface EntryBodySectionProps {
     selector?: string,
 }
 
+export const formatRequest = (body: any, contentType: string, decodeBase64: boolean = true, isBase64Encoding: boolean = false, isPretty: boolean = true): string => {
+    if (!decodeBase64 || !body) return body;
+
+    const chunk = body.slice(0, MAXIMUM_BYTES_TO_FORMAT);
+    const bodyBuf = isBase64Encoding ? atob(chunk) : chunk;
+
+    try {
```

Comment:
I wonder if we need the doubled try-catch both here and in the formatTextBody func..
Make sure we don't print the error twice when there is an error